pocs/utils/images/__init__.py

```diff
@@ -92,44 +98,58 @@ def make_pretty_image(fname, timeout=15, **kwargs):  # pragma: no cover
         return _make_pretty_from_fits(fname, **kwargs)
 
 
-def _make_pretty_from_fits(fname=None, figsize=(10, 8), dpi=150, **kwargs):
+
+def _make_pretty_from_fits(fname=None, figsize=(10, 8), dpi=150, alpha=0.2, pad=3.0, **kwargs):
     header = getheader(fname)
     data = getdata(fname)
     data = focus_utils.mask_saturated(data)
 
     title = kwargs.get('title', header.get('FIELD', 'Unknown'))
     exp_time = header.get('EXPTIME', 'Unknown')
-    filter = header.get('FILTER', 'Unknown filter')
+
+    filter_type = header.get('FILTER', 'Unknown filter')
     date_time = header.get('DATE-OBS', current_time(pretty=True)).replace('T', ' ', 1)
 
     percent_value = kwargs.get('normalize_clip_percent', 99.9)
 
-    title = '{} (Exposure time: {} s, Filter: {}) {}'.format(title, exp_time, filter, date_time)
+    title = '{} (Exposure time: {} s, Filter: {}) {}'.format(title, exp_time, filter_type, date_time)
```

Comment:
Have you looked to see if this gets too long? If so could just have `Super Title (60s H-a)` without spelling out `Exposure time` and `Filter`